components/feature/prompts/src/main/java/mozilla/components/feature/prompts/file/FilePicker.kt

```diff
@@ -159,13 +161,22 @@ internal class FilePicker(
         if (intent?.clipData != null && request.isMultipleFilesSelection) {
             intent.clipData?.run {
                 val uris = Array<Uri>(itemCount) { index -> getItemAt(index).uri }
-                request.onMultipleFilesSelected(container.context, uris)
+                val sanitizedUris = uris.filterUrisUnderPrivateAppDirectory(container.context)
```

Comment:
nit++ (just naming): `filter` retains elements for which the predicate is true. so, `filterUrisUnderPrivateAppDirectory` would suggest that we retain the URIs under the private app directory. Of course, we do the opposite here, we remove (or filter out) the URIs under private app dirs. Maybe `removeUrisUnderPrivateAppDir` or `filterOutUris...`. I think it would make it easier to read.

Reply:
👍